internal/p10k.zsh

```diff
@@ -934,8 +934,8 @@ prompt_anaconda() {
 ################################################################
 # AWS Profile
 prompt_aws() {
-  local aws_profile="${AWS_VAULT:-${AWS_PROFILE:-$AWS_DEFAULT_PROFILE}}"
-  if [[ "$aws_profile" != (default|) ]]; then
```

Comment:
Was this lined changed intentionally? If so, why?

Reply:
This could be considered an optional change, but is actually a safety measure, let me explain.

When you have AWS credentials set in the `default` profile, you are implicitly always logged in. Whenever you execute an `aws` cli command, it automatically uses the default profile when no explicit profile is set. There is a risk you accidentally execute a harmful command while your shell does _not_ show that you have an _active_ profile.

When you do not explicitly set `AWS_DEFAULT_PROFILE=default` in your environment (eg. because you don't use AWS) nothing will change. But when you do set it, it's a way to explicitly show that you do have an active profile, which is important knowledge imho.

Reply:
Never mind, I see that you have them in two commits. It's all good!